tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -373,3 +492,11 @@ Like BaudRate, StopBits, Parity, Databits, HardwareFlowControl.
                 return -1;
         }
 ```
+## Abort parser vs dump parser tool
```

Comment:
This is too Samsung internal specific. Noone understands what abort parser is. Let's remove this from this doc. Could you take two pics of each on confluence? how to execute and the output of the tool. And then write what is different and what is better.